modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/remote/RemoteImageInputStreamSPI.java

```diff
@@ -16,20 +16,21 @@
  */
 package org.geotools.gce.imagemosaic.remote;
 
-import com.sun.imageio.spi.FileImageInputStreamSpi;
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.net.URL;
 import java.util.Locale;
 import javax.imageio.spi.ImageInputStreamSpi;
 import javax.imageio.stream.ImageInputStream;
+
+import it.geosolutions.imageio.stream.input.spi.FileImageInputStreamExtImplSpi;
 import org.geotools.TestData;
 import org.geotools.gce.imagemosaic.RemoteTest;
 
 public class RemoteImageInputStreamSPI extends ImageInputStreamSpi {
 
-    private FileImageInputStreamSpi delegate = new FileImageInputStreamSpi();
+    private FileImageInputStreamExtImplSpi delegate = new FileImageInputStreamExtImplSpi();
```

Comment:
This looks like a sensible replacement: https://github.com/geosolutions-it/imageio-ext/blob/master/library/streams/src/main/java/it/geosolutions/imageio/stream/input/spi/FileImageInputStreamExtImplSpi.java

Only concern would be if imageioext would like a more generic class returned to have flexibility, if so we can change it again in the future.


And has been around since 2010